### PR TITLE
Jp car value

### PR DESCRIPTION
**Pull request by JonoPadoa**

### CODE REVIEW

File: build/controller/valueController.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var __importStar = (this && this.__importStar) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.checkCarValue = void 0;
-const calcService = __importStar(require("../services/calcServices"));
+const calcService = __importStar(require("../services/calcServices.js"));
```
Comment (HMoana) [inline build/controller/valueController.js]: Is function name _importStar compiled from typescript? can the name be changed to something more descriptive?

File: build/server.js [hunk 2]
```diff
@@ -1,16 +1,14 @@
-'use strict'
-var __importDefault =
```
Comment (HMoana) [inline build/server.js]: can "var" be replaced with "let" or "const"?

File: build/server.js [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-const index_1 = __importDefault(require('./routes/index'))
-const server = (0, express_1.default)()
-server.use(express_1.default.json())
-server.use('/api', index_1.default)
-const PORT = 4000
-server.listen(4000, () => {
-  console.log(`server started on port 4000`)
-})
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
```
Comment (HMoana) [inline build/server.js:2]: Is function name _importDefault compiled from typescript? can the name be changed to something more descriptive?

File: coverage/lcov-report/block-navigation.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+var jumpToCode = (function init() {
```
Comment (HMoana) [inline coverage/lcov-report/block-navigation.js:2]: can "var" be changed to "let" or "const"?

File: coverage/lcov-report/sorter.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
+/* eslint-disable */
+var addSorting = (function() {
```
Comment (HMoana) [inline coverage/lcov-report/sorter.js:2]: can "var" be replaced with "let" or "const"?

File: package.json [hunk 6]
```diff
@@ -2,20 +2,19 @@
   "name": "mission-3-apis",
```
Comment (HMoana) [inline package.json:2]: Suggestion to double check packages with collaborator before merge

File: src/test/calcServices.test.ts [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+  const expected = { error: 'Model and year are required.' }
+  let theValue: { error: string } | { carValue: number }
+  // Act
+  theValue = calculateCarValue(input1, input2)
+
+  // Assert
+  expect(theValue).toStrictEqual(expected)
+})
+
+// SAVING THE JS VERSION JUST IN CASE:
```
Comment (HMoana) [inline src/test/calcServices.test.ts:72]: I love comments, but I know the less lines of code to read the better. I understand why you have left this code here. I dont have a solution, more of suggestion. How's about shifting old code that you want to keep to another file? Just a suggestion.

**Review by HMoana — APPROVED** [review]
Clean code, got through a lot of work. Left suggestions only.

File: build/controller/valueController.js (hunk 1 above)
Comment (JonoPadoa) [reply build/controller/valueController.js]: Yea these are the old build files so its compiled JavaScript/  😂

**Review by thetergus — COMMENTED** [review]
LGTM.

**Pull request closed without merging**